src/wp-includes/load.php

```diff
@@ -881,7 +881,7 @@ function wp_start_object_cache() {
 			)
 		);
 
-		wp_cache_add_non_persistent_groups( array( 'counts', 'plugins', 'theme_json' ) );
+		wp_cache_add_non_persistent_groups( array( 'counts', 'plugins', 'theme_json', 'block_template' ) );
```

Comment:
@felixarntz This is the most important part of this PR. This caches can never be a be a transient cache. Files can be edited local or by uploading via FTP. This will trigger no cache invalidation.